### ISSUE TITLE
Make 'notice' issues less alarming

### ISSUE THREAD

**Issue by comnam90** [cstate/cstate#102]
**Is your feature request related to a problem? Please describe.**
Nope

**Describe the solution you'd like**
Currently on the homepage if there is an ongoing issue with the type notice, under incident history it screams at you that there is an unresolved issue. But to me, a notice level issue is non-impacting and shouldn't be red and alarming.

I'd prefer it be a warning with 'In progress' instead. (mock up below)
![Example](https://i.imgur.com/ezwy348.png)

**Describe alternatives you've considered**
I know there are informational posts, however I feel these are more for non-impacting changes or maintenance, as opposed to non-disruptive issues.

**Additional context**
Looking at the theme code, it looks like there are 2 places this needs to be changed.    
 - small.html
 - en.yaml

The end of small.html should be updated to this
```html
    <h3>
      {{ .Title }}
    </h3>

    <!-- Marker -->
    {{ if eq .Params.severity "notice" }}
      <strong class="warning">
        ◆ {{ T "noticeOngoing" }}
      </strong>
    {{ else }}
      <strong class="error">
        {{ if eq .Params.severity "down" }}
        ■ 
        {{ else if eq .Params.severity "disrupted" }}
        ▲ 
        {{ else }}
        ◆ 
        {{ end }}
        {{ T "downtimeOngoing" }}
      </strong>
    {{ end }}
  {{ end }}
</a>
```

And en.yaml (and other translations) should have this added
```yaml
- id: downtimeOngoing
  translation: Unresolved!
- id: noticeOngoing
  translation: In Progress
```

**Comment by comnam90** [comment]
Or even maybe make use of the Informational Icon as well
ℹ

**Comment by mistermantas** [comment]
"In Progress" sounds strange to me (as if the issue is still getting worse or something), but "Unresolved!" might be a bit harsh, too, so I see your point.

I am reluctant to change the strings — or more accurately, add new ones — because I strive for cState to be as correctly translated into as many languages as possible.

As a stopgap, you can change this within your own templates for now.

I guess I could add "This is issue is being worked on right now" or something like that. Not sure though.

**Comment by mistermantas** [comment]
I've implemented this in v4.1.2 without adding any new strings — "This issue is still being resolved" will be used in red like always for downtime affecting issues (down, disrupted) while issues set as a notice will have the 'warning' class so it'll be in yellow/orange. Informational posts are unchanged, I think using the 'i' icon for issues is not a good idea since there's a clear separation between an issue and a post.

Please let me know if you have any other questions or suggestions, always open to hearing them.
